src/App.tsx

```diff
@@ -50,15 +39,15 @@ export const App: React.FC = () => {
           <div className="tile is-parent">
             <div className="tile is-child box is-success">
               <div className="block">
-                <UserSelector value={author} onChange={setAuthor} />
+                <UserSelector />
               </div>
 
               <div className="block" data-cy="MainContent">
                 {!author && <p data-cy="NoSelectedUser">No user selected</p>}
 
-                {author && !loaded && <Loader />}
+                {author && loading && <Loader />}
```

Comment:
If it describes a loading state it should be boolean and if it boolean it should start from `is` or `has`. Change it for all the cases
and u can improve ur code style for better readability
```suggestion
                {(author && isLoading) && (
                  <Loader />
                )}
```